opencti-platform/opencti-front/lang/front/fr.json

```diff
@@ -1384,6 +1384,7 @@
   "Graph": "Graph",
   "Graph type": "Type de graphique",
   "Graph view": "Vue graphe",
+  "GraphQL playground": "Terrain de jeu GraphQL",
```

Comment:
should we just use "Playground"? "Terrain de jeu" seems weird
![image](https://github.com/user-attachments/assets/49c73d02-af0a-4596-be56-101a27242648)
well  this is not a big deal

Reply:
fixed :)